apps/mdn/mdn-aws/infra/versions.tf

```diff
@@ -1,10 +1,16 @@
-
 terraform {
-  required_version = ">= 0.13"
+  backend "s3" {
```

Comment:
just a note, it seems like we're not consistently naming the file this block lives in, I wonder if it's worth having a team discussion about what file should backend blocks go, what file should the required versions go, where should the providers be defined?  I've seen versions.tf/state_config.tf/state.tf/providers.tf